ksql-rest-model/src/main/java/io/confluent/ksql/rest/entity/HostStatusEntity.java

```diff
@@ -62,6 +60,11 @@ public void setLastStatusUpdateMs(final long lastStatusUpdateMs) {
     this.lastStatusUpdateMs = lastStatusUpdateMs;
   }
 
+  public void setPerQueryActiveStandbyEntity(
+      final Map<String, ActiveStandbyEntity> perQueryActiveStandbyEntity) {
+    this.perQueryActiveStandbyEntity = perQueryActiveStandbyEntity;
+  }
```

Comment:
Can we make this type immutable please?

Change this to a `withPerQueryActiveStandbyEntity` that returns a new instance.